Phonebook.Frontend/src/app/shared/directives/feedback-drawer/feedback-drawer-sheet/feedback-drawer-sheet.component.html

```diff
@@ -1,8 +1,26 @@
 <mat-list>
-  <h3 mat-line i18n="FeedbackDrawerSheetComponent|Title of the Feedback Drawer@@FeedbackDrawerSheetComponentTitle">
-    Leave Feedback, report a Bug or submit a new Idea
+  <h3
+    mat-line
+    i18n="FeedbackDrawerSheetComponent|Title of the Feedback Drawer@@FeedbackDrawerSheetComponentTitle"
+    class="pb-text-align-center"
+  >
+    Leave Feedback, report a Bug or suggest a new Idea
   </h3>
-
+  <mat-list-item *ngIf="runtimeEnvironment.contactEmail">
+    <a
+      mat-button
+      color="primary"
+      target="_blank"
+      rel="noopener"
+      href="{{ runtimeEnvironment.contactEmail }}"
+      (click)="closeBottomSheet()"
+    >
+      <span
+        i18n="FeedbackDrawerSheetComponent|Button for contacting the private Organization via email@@FeedbackDrawerSheetComponentButtonPrivateOrganizationMail"
+        >Contact your private organization via email</span
```

Comment:
```suggestion
        >Contact a person at your organization</span
```
Maybe this makes it differ even more from the other button?

Reply:
I guess youre right, but usually you are not contacting a single person but a collective e-mail pool, or what do you think?

Reply:
> I guess youre right, but usually you are not contacting a single person but a collective e-mail pool, or what do you think?

Sure, your do. But this should not be a problem :)

Reply:
Mhm, Up to you. I don't like the option two options altogether. From the user pov there will be the question: "what's the difference?"

Reply:
Maybe we should generally think about removing the second link to the repo? The user can't realy contact the project via the repo, can he? If a user wants to report a bug or feature he has the other two links. Or we rename the link to "take a look at the repository on Github" or sth similar?

Reply:
On the link to the private repo he can open a new internal issue (e.g. user authentification, wrong roomplan or something like). It prevents the case that people leak internal things in an official github repo.

If you mean the different GitHub options i am also not sure if we need both or can combine.. but for now it's the easiest solution for not so technical people.

Reply:
I only meant the private organisation links/mail. Maybe we should remove the internal link or replace it by one function to contact the somebody in the company.

Reply:
I would say let us removing the internal issue link and let the other option like this:
- Contact Email
- New Idea
- Report bug
- cancel

The internal issue tracker was most of the time not so useful so let us remove the link.

Reply:
That's what I meant, but I would prefer the solution of displaying the issue tracker link if supplied by the company. Because if you have an issue tracker nobody wants to be mailed :)

Reply:
So you mean we leave the implementation like this, but remove the contactUrl in our deployment?

Reply:
@DanielHabenicht how should we proceed?

Reply:
@DanielHabenicht please answer before i approve :)

Reply:
Leave as it is

Reply:
Or implement one button which shows the contact email, or the issue tracker link, if it is provided.